selinon/global_config.py

```diff
@@ -130,7 +130,8 @@ def _parse_trace_sentry(cls, trace_def):
         if unknown_conf:
             raise ConfigurationError("Unknown configuration for Sentry trace function supplied: %s" % unknown_conf)
 
-        cls._trace_sentry.append(trace_def.get('dsn', True))
+        dsn = trace_def.get('dsn', True).format(**os.environ)
```

Comment:
I have resolved it in a different manner. I have changed the default value to the empty string `''` instead of a boolean. I hope you like it this way.